README.md

```diff
@@ -29,6 +29,22 @@ See [mbed_lib.json](mbed_lib.json) for all configurable options.
 
 See [test README](TESTS/mqtt/README.md) to find out about tests-specific configuration configuration.
 
+### Building with CLI V2
+
+Assuming you have this repository cloned or somehow available at the `<your-project-root>/mbed-mqtt` directory.
+
+Add to your project's `CMakeLists.txt` file the following lines:
+
+```
+add_subdirectory(mbed-mqtt)
+
+target_link_libraries(${APP_TARGET} mbed-mqtt)
```

Comment:
group them together to one `target_link_libraries()`

Reply:
Okay, I'll do it.

Reply:
Done. I've declared all dependencies explicitly, so this list has shrunk to just single entry `mbed-mqtt`.